packager/media/base/fourccs.h

```diff
@@ -59,6 +60,7 @@ enum FourCC : uint32_t {
   FOURCC_dvvC = 0x64767643,
   FOURCC_ec_3 = 0x65632d33,  // "ec-3"
   FOURCC_ec3d = 0x65633364,
+  FOURCC_ac_4 = 0x61632d34,  // "ac-4"
```

Comment:
ditto. Can you move it after FOURCC_ac_3?

Reply:
Sure.